salt/modules/zypper.py

```diff
@@ -628,16 +628,16 @@ def install(name=None,
         log.info('Targeting repo {0!r}'.format(fromrepo))
     else:
         fromrepoopt = ''
+    cmd_install = ['zypper', '--no-refresh', '--non-interactive', 'install',
```

Comment:
Here `--no-refresh` is hardcoded. Zypper acts according to the configuration, so if it does the refresh, then it is for a reason. How to turn on refresh in this case? Instead, I would make it configurable, not just hardcode commands.